### ISSUE TITLE
Identify axis

### ISSUE THREAD

**Issue by zebart00** [ekapope/Combine-CSV-files-in-the-folder#1]
Beautiful code! Thanks!

Your code appended my columns.
I resolved it with:
combined_csv = pd.concat([pd.read_csv(f) for f in all_filenames ],axis=1)

**Comment by pytholic** [comment]
yeah i had same issue and suggested this, but you already suggested it.
